Userland/Libraries/LibGUI/AbstractView.cpp

```diff
@@ -759,13 +761,19 @@ void AbstractView::drag_move_event(DragEvent& event)
 {
     if (!model())
         return;
+
     auto index = index_at_event_position(event.position());
     ModelIndex new_drop_candidate_index;
-    if (index.is_valid()) {
-        bool acceptable = model()->accepts_drag(index, event.mime_types());
-        if (acceptable)
-            new_drop_candidate_index = index;
+    bool acceptable = model()->accepts_drag(index, event.mime_types());
+
+    if (acceptable && index.is_valid())
+        new_drop_candidate_index = index;
```

Comment:
My reasoning here is that we can accept a drag event with an invalid index, however the drop candidate will not be updated since there is none. This allow scrolling to happen even when we're not on top of a drop candidate